packaging/android/build.sh

```diff
@@ -291,6 +292,8 @@ if [ "$QUICK" = "" ] ; then
 			-DCMAKE_INSTALL_LIBDIR="lib" \
 			-DBUILD_SHARED_LIBS=OFF \
 			-DCMAKE_DISABLE_FIND_PACKAGE_BZip2=TRUE \
+			-DZLIB_VERSION_STRING=1.2.7 \
```

Comment:
is hard coding the version number here really a good idea?

Reply:
It really doesn't matter what version you enter, as long its above 1.1.2. 1.2.7 is the version thats shipped in the android NDK, and i don't think thats about to change because its a part of the stable NDK apis.

The `FIND_PACKAGE(ZLIB REQUIRED)` can't find zlib in the NDK, so thats why I'm convincing libzip build system that its just there.